src/__tests__/App.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
+import App from '../App';
+
+import '@testing-library/jest-dom';
+
+describe('App component', () => {
+  it('should render the form inputs and button', () => {
+    render(<App />);
+
+    expect(screen.getByLabelText(/Number of Floors/i)).toBeInTheDocument();
+    expect(screen.getByLabelText(/Number of Lifts/i)).toBeInTheDocument();
+    expect(screen.getByRole('button', { name: /Start Simulation/i })).toBeInTheDocument();
+  });
+
+  it('should show an error message for invalid input values', () => {
```

Comment:
can you please add test cases for below the given range as well?

Reply:
have added in the last PR